pims_nd2/ND2SDK.py

```diff
@@ -8,8 +8,9 @@
     nd2 = cdll.LoadLibrary(os.path.join(os.path.dirname(__file__), 'ND2SDK',
                                         'linux', 'libnd2ReadSDK.so'))
 elif platform == "darwin":
-   raise OSError("Unsupported OS. The ND2SDK for OSX is included, "
-                 "please try to implement!")
+    dlldir = os.path.join(os.path.dirname(__file__), 'ND2SDK', 'osx', 'nd2sdk.framework', 'versions', '1')
```

Comment:
does it also work if you leave out the folders `version` and `1`? there is also an `nd2sdk` in `nd2sdk.framework`.